pkg/machine/wsl/machine.go

```diff
@@ -830,7 +830,15 @@ func launchWinProxy(v *MachineVM) (bool, string, error) {
 		return globalName, "", err
 	}
 
-	dest := fmt.Sprintf("ssh://root@localhost:%d/run/podman/podman.sock", v.Port)
+	destSock := "/run/user/1000/podman/podman.sock"
```

Comment:
should we hardcode 1000 here or should it be `rootless.GetRootlessUID()`?

Reply:
In this case the UID is referring to a fixed configuration for the Fedora distro that podman machine controls, so regardless of system configuration the guest will always have a uid of 1000 (vs rootless.GetRootlessUID which return the host value). Granted a constant value in the code would have been cleaner.